findoutlie/detectors.py

```diff
@@ -51,3 +53,20 @@ def iqr_detector(measures, iqr_proportion=1.5):
     # * You'll likely need np.logical_or
     # https://textbook.nipraxis.org/numpy_logical.html
     # +++your code here+++
+
+
+    Q1 = np.percentile(measures, 25, interpolation='midpoint')
+    Q2 = np.percentile(measures, 50, interpolation='midpoint')
+    Q3 = np.percentile(measures, 75, interpolation='midpoint')
+    IQR= Q3-Q1
+
+    outlier= []
+
+    for i in range(len(measures)):
+        if ( (measures[i]>(Q3+IQR*iqr_proportion)) | (measures[i]<(Q1-IQR*iqr_proportion))  ):
+            outlier.append(i)
```

Comment:
@Nauman702 you don't have to return the indices of the outlier rather you have to return a boolean array where the indices should have a value "True" for outliers. This is why you are getting the output as mentioned in #18